stac_ipyleaflet/core.py

```diff
@@ -341,7 +344,7 @@ def find_layer(self, name: str):
         for layer in layers:
             if layer.name == name:
                 return layer
-        return None
+        return
```

Comment:
This looks like a good example that would violate the consistency of return statements (mixing a `return layer` with an empty `return`).